gatsby-config.js

```diff
@@ -83,6 +83,31 @@ module.exports = {
     },
     'gatsby-plugin-catch-links',
     'gatsby-plugin-sitemap',
+    {
+      resolve: 'gatsby-plugin-robots-txt',
+      options: {
+        host: config.siteUrl + pathPrefix,
+        sitemap: config.siteUrl + pathPrefix + 'sitemap.xml',
```

Comment:
Is there value in overriding the defaults here? It looks like these should resolve to `gatsby-plugin-sitemap`'s defaults.

https://www.gatsbyjs.org/packages/gatsby-plugin-robots-txt/#options

I'm guessing the benefit would be the ability to append the pathPrefix?

Reply:
I guess the benefit is not to make the deploy previews from netlify indexable. 
Since the `pathPrefix` is `/` in config, I could omit it here and the result would be the same.